WhatsApp/app/src/main/java/com/example/whatsapp/ui/fragments/settings/notifications/NotificationsFragment.kt

```diff
@@ -1,19 +1,250 @@
 package com.example.whatsapp.ui.fragments.settings.notifications
 
+import android.app.AlertDialog
+import android.app.Notification
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.content.Context
+import android.content.Context.NOTIFICATION_SERVICE
+import android.graphics.Color
+import android.os.Build
 import android.os.Bundle
-import androidx.fragment.app.Fragment
+import android.view.HapticFeedbackConstants
+import android.view.LayoutInflater
+import android.view.View
+import androidx.core.content.ContextCompat.getSystemService
 import androidx.navigation.NavController
 import com.example.whatsapp.R
+import com.example.whatsapp.base.BaseFragment
+import com.example.whatsapp.databinding.FragmentNotificationsBinding
 import dagger.hilt.android.AndroidEntryPoint
 
+
```

Comment:
ctrl + alt + o and ctrl + alt + l to remove unused imports and reformat

Reply:
done :)